docs/source/installation.rst

```diff
@@ -0,0 +1,68 @@
+.. _installation:
+
+Installation
+============
+
+Prerequisites
+-------------
+
+- Python 3.9 or newer
+
+We recommend installing urlwatch in an isolated environment (virtualenv or
+pipx) to avoid interfering with other Python packages.
+
+Install with pip
+----------------
+
+Use pip inside an existing virtual environment (or add ``--user`` if you
+prefer a user-local install):
+
+.. code-block:: bash
+
+    python -m pip install urlwatch
+
+Install with pipx
+-----------------
+
+If you want urlwatch available as a standalone CLI without managing a
+virtual environment yourself:
+
+.. code-block:: bash
+
+    pipx install urlwatch
+
+Upgrade an existing installation
+--------------------------------
+
+To update to the latest release from PyPI:
+
+.. code-block:: bash
+
+    python -m pip install --upgrade urlwatch
```

Comment:
This is the only command that we should document, it should work for both first-time installs and upgrades. Let's not confuse the users with pipx and definitely don't let beginner users install from Git.